### PR TITLE
update formatting of dao details

### PR DESCRIPTION
**Pull request by mohamedamer453**
### Description

Update the formatting of the dao page details.

image

### Changes

- Updated the formatting and removed the list item.
- Added more info to the message.

### Related Issues

#1492 

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by maayarosama — APPROVED** [review]

File: packages/playground/src/dashboard/dao_view.vue [hunk 1]
```diff
@@ -218,9 +218,11 @@
                 >Votes are weighted based on the farmers stake in the network. One vote by default is 1 weight.</span
               >
               <span> If the farmers has nodes, the weight of the vote is calulcated as following:</span>
-              <ul>
-                <li>Sum of all nodes of the farmer: (node CU * 2 + node SU)</li>
-              </ul>
+              <span
+                ><b>
+                  The farmer's vote weight is equal to the sum of all nodes of the farmer: (node CU * 2 + node SU)</b
+                ></span
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:224]: maybe "The farmer's vote weight is the sum of all the farmer's nodes weight. A node's weight is calculated as: node CU * 2 + node SU." ?

Comment (mohamedamer453) [reply packages/playground/src/dashboard/dao_view.vue:226]: updated in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1551/commits/e69ecc7008aa43ebd3075f4b56074fbd1b1ddce0

**Review by zaelgohary — COMMENTED** [review]
Good job! Just one remark, I think we need to capitalize "information" in the title and maybe add some space after the first paragraph.

https://github.com/threefoldtech/tfgrid-sdk-ts/blob/e69ecc7008aa43ebd3075f4b56074fbd1b1ddce0/packages/playground/src/dashboard/dao_view.vue#L200C55-L200C55

**Comment by mohamedamer453** [comment]
> Good job! Just one remark, I think we need to capitalize "information" in the title and maybe add some space after the first paragraph.
> 
> https://github.com/threefoldtech/tfgrid-sdk-ts/blob/e69ecc7008aa43ebd3075f4b56074fbd1b1ddce0/packages/playground/src/dashboard/dao_view.vue#L200C55-L200C55

updated in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1551/commits/4372b75d045ebcd118957f03cbc12b7729b89fe4

image

**Pull request merged**